scripts/zephyr_module.py

```diff
@@ -233,7 +233,8 @@ def main():
         from west.util import WestNotFound
         try:
             manifest = Manifest.from_file()
-            projects = [p.posixpath for p in manifest.get_projects([])]
+            projects = [p.posixpath for p in manifest.get_projects([])
```

Comment:
Why do we have this change here?

Reply:
It was a mistake, but I actually couldn't get west going AT ALL after rebasing to master without that change. I'll get rid of it.

Reply:
I actually still can't compile without it (using a custom west.yml), but I'll revert it and solve it separately.